Header header

# The number of missed headers that the channel has observed
uint64 num_missed_headers

# The number of dropped UDP assemblers that the channel has observed
uint64 num_dropped_assemblers

# The number of image metadata messages were received
uint64 num_image_meta_data

# The number of images that were dispatched
uint64 num_dispatched_image

# The number of lidar scans that were dispatched
uint64 num_dispatched_lidar

# The number of dispatched PPS messages
uint64 num_dispatched_pps

# The number of dispatched IMU messages
uint64 num_dispatched_imu

# The number of dispatched compressed images
uint64 num_dispatched_compressed_image

# The number of dispatched ground surface spline events
uint64 num_dispatched_ground_surface_spline

# The number of dispatched april tag detection events
uint64 num_dispatched_april_tag_detections


